src/services/expenses.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+const expenses = [];
+
+let countExpensesId = 1;
+
+const getExpenses = ({ from, to, category, userId }) => {
+  const startFrom = Date.parse(from);
+  const end = Date.parse(to);
+  let filteredExpenses = expenses;
+
+  if (startFrom) {
+    const result = filteredExpenses.filter(expense => {
+      const date = Date.parse(expense.spentAt);
+
+      if (date >= startFrom) {
+        return expense;
+      }
+    });
+
+    filteredExpenses = result;
+  }
+
+  if (end) {
+    const result = filteredExpenses.filter(expense => {
+      const date = Date.parse(expense.spentAt);
+
+      if (date <= end) {
+        return expense;
+      }
+    });
+
+    filteredExpenses = result;
+  }
+
+  if (category) {
+    const result = filteredExpenses.filter(expense => (
+      expense.category === category)
+    );
+
+    filteredExpenses = result;
+  }
+
+  if (userId) {
+    const result = filteredExpenses.filter(expense => (
+      expense.userId === +userId
+    ));
+
+    filteredExpenses = result;
+  }
+
+  return filteredExpenses;
+};
+
+const getExpensesById = (expenseId) => {
+  const foundExpense = expenses.find(expense => expense.id === expenseId);
+
+  return foundExpense || null;
+};
+
+const createExpense = ({
+  userId,
+  spentAt,
+  title,
+  amount,
+  category,
+  note = 'Any notes',
```

Comment:
Don't set initial note here, let it be null
What if we'll handle this case on FE part and show a different message?